### PR TITLE
change to the contact page

### PR DESCRIPTION
**Pull request by frnega**
in this project i added a contact page

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -166,5 +165,44 @@ <h4>Multi-Post Stories</h4>
 <br>
 <button>See Project</button>
 </section>
+<section class="contact-section">
+	<h6>CONTACT US</h6>
+	<p id="three">If you have an application you are 
+		interested in developing, a feature that you
+		 need built or a project that needs
+		  coding.I’d love to help with it</p>
+			<form action="/conatact-form-page" method="post">
```
Comment (Bondok6) [inline index.html:174]: - [ ] According to the requirements you should use the [Formspree service](https://formspree.io/html) to collect the data
- action value should be similar to https://formspree.io/f/{form_id}, so kindly fix that 👌

File: index.html [hunk 2]
```diff
@@ -166,5 +165,44 @@ <h4>Multi-Post Stories</h4>
...
+<footer>
+<section class="headline3">
+	<a href="#" class="fa fa-facebook"></a>
+	<a href="#" class="fa fa-twitter"></a>
+	<a href="#" class="fa fa-linkedin"></a>
+	<a href="#" class="fa fa-github"></a>
+	<a href="#" class="fa fa-medium"></a>
+	<p>fr.nega@gmail.com</p>
+</section> 
+</footer>
```
Comment (Bondok6) [inline index.html:206]: - [ ] According to the requirements You must stick to the design as much as possible (e.g., font, colors, images, text, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=23%3A10). So kindly decrease the spaced between `<h6>` and the `<p>` tags, center the submit button, change the color of the horizontal line, and use `text-decoration: none` in `<a>` tags to remove the underline in your social icons 💯👌

| Your Design  | Figma Design  |   
| --- | --- |
|   |  |

**Review by Bondok6 — CHANGES_REQUESTED** [review]
## Hi Team, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
- No Linter errors ✅
- Professional repo ✅

## Required Changes ♻️
- [ ] _Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by frnega** [comment]
made a change to the review

File: index.html (hunk 2 above)
Comment (sja-thedude) [reply index.html]: Implemented 👍

File: index.html (hunk 1 above)
Comment (sja-thedude) [reply index.html]: Implemented 👍

**Pull request merged**